src/Application.cc

```diff
@@ -306,6 +306,7 @@ bool Application::LoadPlugin(const std::string &_filename,
 
   common::SystemPaths systemPaths;
   systemPaths.SetPluginPathEnv(this->dataPtr->pluginPathEnv);
+  systemPaths.AddPluginPaths("./" + std::string(IGN_GUI_PLUGIN_PATH));
```

Comment:
Can you explain what's the intention here? `IGN_GUI_PLUGIN_PATH` doesn't exist and this fails to compile. This also doesn't seem to be platform-independent.